test/e2e/framework/metrics/kubelet_metrics.go

```diff
@@ -29,13 +29,26 @@ import (
 	"k8s.io/apimachinery/pkg/util/sets"
 	clientset "k8s.io/client-go/kubernetes"
 	"k8s.io/component-base/metrics/testutil"
-	dockermetrics "k8s.io/kubernetes/pkg/kubelet/dockershim/metrics"
-	kubeletmetrics "k8s.io/kubernetes/pkg/kubelet/metrics"
 	e2elog "k8s.io/kubernetes/test/e2e/framework/log"
 )
 
 const (
 	proxyTimeout = 2 * time.Minute
+	// dockerOperationsLatencyKey is the key for the operation latency metrics.
+	// Taken from k8s.io/kubernetes/pkg/kubelet/dockershim/metrics
+	dockerOperationsLatencyKey = "docker_operations_duration_seconds"
+	// Taken from k8s.io/kubernetes/pkg/kubelet/metrics
+	kubeletSubsystem = "kubelet"
+	// Taken from k8s.io/kubernetes/pkg/kubelet/metrics
+	podWorkerDurationKey = "pod_worker_duration_seconds"
+	// Taken from k8s.io/kubernetes/pkg/kubelet/metrics
+	podStartDurationKey = "pod_start_duration_seconds"
+	// Taken from k8s.io/kubernetes/pkg/kubelet/metrics
+	cgroupManagerOperationsKey = "cgroup_manager_duration_seconds"
+	// Taken from k8s.io/kubernetes/pkg/kubelet/metrics
+	podWorkerStartDurationKey = "pod_worker_start_duration_seconds"
+	// Taken from k8s.io/kubernetes/pkg/kubelet/metrics
+	pLEGRelistDurationKey = "pleg_relist_duration_seconds"
```

Comment:
nit: I guess the first "p" is for local, but I feel `plegRelistDurationKey` seems more readable.

Reply:
/cc @oomichi

Reply:
Done.